enabler/src/main/java/com/openxc/enabler/OpenXcEnablerActivity.java

```diff
@@ -82,7 +82,11 @@ public void onCreate(Bundle savedInstanceState) {
     @Override
     protected void onSaveInstanceState(Bundle outState) {
         super.onSaveInstanceState(outState);
-        outState.putInt("tab", mPager.getCurrentItem());
+        try {
+            outState.putInt("tab", mPager.getCurrentItem());
+        }catch (NoClassDefFoundError e){
+            Log.w(TAG, "Busgnag is unsupported when building from Eclipse", e);
```

Comment:
You probably should have a better error message.  Eclipse has long been deprecated and it seems like there may be an exception generated from getting the current item from a non-existent or bad pager control.